modules/my_preferences/README.md

```diff
@@ -0,0 +1,29 @@
+# My Preferences
+
+## Purpose
+
+This module allows a user to edit their basic profile information as well as
+customize their notification settings.
+
+## Intended Users
+
+Every user is able to modify their profile information.
+
+## Scope
+
+This module allows a user to edit their basic profile information as well as
+customize their notification settings.
+
+## Permissions
+
+A user is always able to access the My Preferences page.
+
+## Configurations
+
+Some settings related to email notifications affect this module.
+
+## Interactions with LORIS
+
+- Modifications performed on the My Preferences page will be reflected on 
+the user account page when viewing the edited account.
```

Comment:
I don't think that's really an interaction worth noting. Any data modified anywhere should be reflected other places that use the data..